DOC: Fix selector expression documentation (#6050)
nextflow-io/nextflow: docs/config.md

}
```

> The above configuration snippet requests 2 cpus and 4 GB of memory for processes labeled as `foo` and processes labeled as `bar`.

Reviewer: ```suggestion
The above configuration snippet requests 2 cpus and 4 GB of memory for processes labeled as `foo` or `bar`.
```

> The above configuration snippet sets 2 cpus and 4 GB of memory to the processes annotated with a label `foo` or with a label `bar`.

Reviewer 2: ```suggestion
The above configuration snippet requests 2 cpus and 4 GB of memory for processes labeled as `foo` and processes labeled as `bar`.
```
